agent/src/main/bin/init.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# ----------------------------------------------------------------------------
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+echo "wsEndpoint=$1" > /opt/testgrid/agent-config.properties
+echo "region=$2" >> /opt/testgrid/agent-config.properties
+echo "testPlanId=$3" >> /opt/testgrid/agent-config.properties
+echo "provider=$4" >> /opt/testgrid/agent-config.properties
```

Comment:
Ideally, we should parse the input arguments via [getopts](https://www.mkssoftware.com/docs/man1/getopts.1.asp). Shall we add a TODO item within the file itself and/or raise a ticket, so we know about the limitation? :)

Reply:
ATM, init.sh call inject into User Data section of the CF by TG. So will add TODO here.